doc/2/framework/classes/plugin-context-constructors/properties/index.md

```diff
@@ -92,23 +92,23 @@ const req = new context.constructors.KuzzleRequest(request, {
 const user = await context.accessors.execute(req) 
 ```
 
-## `StorageClient`
+## `Mutex`
+
+Instantiates a new mutex, allowing to lock a resource. Works both with single-node and cluster environments.
 
-Constructor for [Elasticsearch SDK Client](https://www.elastic.co/guide/en/elasticsearch/client/javascript-api/current/api-reference.html).
+See [Mutex documentation](/core/2/framework/classes/mutex) for more information.
 
-## Usage
+### Arguments
```

Comment:
You can just redirect on the original Mutex page (the ctor have arguments actually)